packages/common/index.ts

```diff
@@ -12,6 +12,9 @@ import { Type } from "class-transformer";
 import "reflect-metadata";
 
 export const PROD_URL = "https://khouryofficehours.com";
+export const isProd = (): boolean =>
+  process.env.DOMAIN === PROD_URL ||
+  (typeof window !== "undefined" && window?.location?.origin === PROD_URL);
```

Comment:
I don't see anything wrong with this code, however my javascript knowledge is limited so I don't feel too qualified to speak on this. 

Even though it does seem a little sketchy to be checking `isProd` for both the client and server in one function, I like the simplicity of it.